Marlin/UBL_G29.cpp

```diff
@@ -21,7 +21,7 @@
  */
 
 #include "Marlin.h"
-#if ENABLED(AUTO_BED_LEVELING_UBL)
+#if ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(UBL_MESH_EDIT_ENABLED)

```

Comment:
The existence of G29 will depend on both `AUTO_BED_LEVELING_UBL` and `UBL_MESH_EDIT_ENABLED`? I think `G29` should be available just with `AUTO_BED_LEVELING_UBL` even though it is technically a way to "edit" the mesh.